crates/re_viewer_context/src/gpu_bridge/colormap.rs

```diff
@@ -64,40 +64,63 @@ fn colormap_preview_ui(
     Ok(response)
 }
 
+fn colormap_variant_ui(
+    render_ctx: Option<&re_renderer::RenderContext>,
+    ui: &mut egui::Ui,
+    option: &re_types::components::Colormap,
+    map: &mut re_types::components::Colormap,
+) -> egui::Response {
+    let list_item = list_item::ListItem::new().selected(option == map);
+
+    let response = if let Some(render_ctx) = render_ctx {
+        list_item.show_flat(
+            ui,
+            list_item::PropertyContent::new(option.to_string())
+                .min_desired_width(150.0)
+                .value_fn(|ui, _| {
+                    if let Err(err) = colormap_preview_ui(render_ctx, ui, *option) {
+                        re_log::error_once!("Failed to paint colormap preview: {err}");
+                    }
+                }),
+        )
+    } else {
+        list_item.show_flat(ui, list_item::LabelContent::new(option.to_string()))
+    };
+
+    if response.clicked() {
+        *map = *option;
+    }
```

Comment:
I think this is the culprit: clicked is the wrong thing to check for

Reply:
someone tested their PR properly it seems... 🙄👏🏻 🤦🏻 

Actually, `clicked()` is the right thing here, but the returning `Response` must be `marked_changed` to be picked up by the edit ui machinery. Fixed now.